tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/AnonymousClosedRecordTest.java

```diff
@@ -75,6 +75,11 @@ public void anonymousRecordWithTypeInclusion() {
         BRunUtil.invoke(compileResult, "anonymousRecordWithTypeInclusion");
     }
 
+    @Test(description = "Test record type with binding pattern var in default value", enabled = false)
```

Comment:
Do we have a comment and a separate issue to track on why this test being disabled?

Reply:
It was a mistake